ui/components/useSEO/index.tsx

```diff
@@ -0,0 +1,30 @@
+import { useMetaTag, useTitle } from 'ui/components/HeadProvider'
+
+export * from './JSON-LD'
+
+export function useKeywords(keywords: string[]) {
+  return useMetaTag({ name: 'keywords', content: keywords.join(',') })
+}
+
+export function useDescription(desc: string) {
+  return useMetaTag({ name: 'description', content: desc })
+}
+
+export function useNoIndex(index: boolean) {
+  useMetaTag({ name: 'robots', content: index ? 'index,follow' : 'noindex,nofollow' })
+  useMetaTag({ name: 'googlebot', content: index ? 'index,follow' : 'noindex,nofollow' })
+}
+
+interface SEOProps {
+  title?: string
+  description?: string
+  keywords?: string[]
+  noindex?: boolean
+}
+
+export function useSEO(props: SEOProps) {
```

Comment:
This is a misuse of the term `SEO`. It stands for Search Engine Optimization and is a while art/discipline itself, rather than being a handful of meta-tags. Please change.

Reply:
Maybe better to just skip the rollup hook? Doesn't seem to make things much cleaner for the components using it.